templates/webserver/webserver-deployment.yaml

```diff
@@ -59,15 +59,15 @@ spec:
         - name: wait-for-airflow-migrations
           image: {{ template "airflow_image" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
-          args: ["airflow-migration-spinner", "--timeout=60"]
+          args: ["db", "check-migrations", "--migration-wait-timeout", "60"]
```

Comment:
The airflow-chart needs to support all astronomer certified images. Do these modifications to args work with all published AC versions?

Reply:
@sjmiller609 there isn't any non horrible way we can create a chart that both works for 1.10 and 2.0. Maybe we need to fork this repo?

Reply:
To merge into master it needs to work with master, then we can cherry-pick and fix it for the 1-10-test branch

Reply:
@schnie can you decide what to do for this one? The only item I wish to protect in this case is AC compatibility on the master and release branches of this repository.

Reply:
I'm cool with us forking "astronomer/oss-airflow-chart" and then donating that. Would that work?

Reply:
I'll let greg speak to what pattern he wants, I think a branch on this repo seems reasonable, the benefit as opposed to a fork being that CI will keep working without further modification to CI configuration.

my suggestion:

- branch on this repo, add branch protection rules in admin settings of this repository
- when it is donation-ready, then copy it into greg's personal fork of airflow
- Commit to include all current authors of the airflow chart
- PR to apache/airflow